### DISCUSSION TITLE
No .csv file

### DISCUSSION THREAD

**Discussion by taylordthegreat** [davetapley/helium-tax#90] [General]
Hi, when I use the tool to calculate my HNT transactions I'm never prompted to download a .csv file. It simply stops counting at some point and nothing else happens. Am I missing something or is tool not working well for me? Thanks ahead of time!

**Comment by davetapley** [comment]
You're probable experiencing this:
- https://github.com/davetapley/helium-tax/issues/79

All I can recommend is coming back and trying again in a few hours, others have had luck that way.

---

If you're comfortable with (or want to learn!) [dev tools](https://developer.chrome.com/docs/devtools/) **and** you don't see 429s in the network tab, then please drop a screenshot of the dev tools console here if there are other errors 🙏🏻
